include/glow/Backends/LayoutConverter.h

```diff
@@ -31,7 +31,7 @@ Node *convertConvToNCHWConv(ConvolutionNode *CN, Function *F) {
 
   auto dimsNHWC = ShapeNHWC(CN->getResult().getType()->dims());
   auto dimsNCHW = {dimsNHWC.n, dimsNHWC.c, dimsNHWC.h, dimsNHWC.w};
-  auto outTy = F->getParent()->uniqueType(ElemKind::FloatTy, dimsNCHW);
+  auto outTy = F->getParent()->uniqueTypeWithNewShape(CN->getType(0), dimsNCHW);
```

Comment:
nit: CN->getResult().getType() should be good.

Reply:
Fixed.